torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -1349,13 +1356,18 @@ def try_save_cache_entry(  # noqa: F811
                     backward_time_taken_ns,
                     sanitized_aot_config=sanitize_aot_config(aot_config),
                     guards_expr=guards_expr,
+                    cached_lazy_backward_info=CachedAutogradLazyBackwardCompileInfo(
```

Comment:
Nit: instead of clearing the meta fields off the bw_module in try_save_cache_entry, it would be better if `CachedAutogradLazyBackwardCompileInfo.__init__` automatically cleared the meta fields from the bw_module before saving, so that it's not possible to accidentally create a CachedAutogradLazyBackwardCompileInfo that is unserializable if someone decides to make one in a different code path